src/main/kotlin/extensions/IntegrationsExtentions.kt

```diff
@@ -29,12 +29,15 @@ suspend fun Novu.providerWebhook(providerId: String): ResponseWrapper<Boolean>?
     return response.extractResponse(logger)
 }
 
-suspend fun Novu.updateIntegration(integrationId: String, request: IntegrationRequest): ResponseWrapper<IntegrationReponse>? {
+suspend fun Novu.updateIntegration(integrationId: String, request: IntegrationRequest): ResponseWrapper<IntegrationResponse>? {
     val response = integrationsApi.updateIntegration(integrationId, request)
     return response.extractResponse(logger)
 }
 
-suspend fun Novu.deleteIntegration(integrationId: String): ResponseWrapper<List<IntegrationReponse>>? {
+suspend fun Novu.deleteIntegration(integrationId: String): ResponseWrapper<List<IntegrationResponse>>? {
     val response = integrationsApi.deleteIntegration(integrationId)
     return response.extractResponse(logger)
 }
+
+suspend fun Novu.setPrimaryIntegration(integrationId: String): ResponseWrapper<IntegrationResponse>? =
+    integrationsApi.setPrimaryIntegration(integrationId).extractResponse(logger)
```

Comment:
We've not confirmed that this would work fine especially because it's a suspend function
So please put each declaration on a different line just like the other functions in this class

Reply:
It works, but I'll make it similar to other functions